feature-flags/README.md

```diff
@@ -6,17 +6,20 @@ of 3 categories:
 
 1. Release Toggle - This is a feature flag which is used to allow in progress
    features to be merged into the main branch and go out to production "dark".
-2. Ops Toggles - A feature flag designed to allow us to incrementally release a
-   feature to production, typically targetted at the center level.
-3. Configuration Toggles - These are flags that enable / disable features on a
+   These are managed in the Feature Flag section of Administrate.
+3. Ops Toggles - A feature flag designed to allow us to incrementally release a
+   feature to production, typically targetted at the center level. These are managed
+   in the Feature Flag section of Administrate.
+4. Configuration Toggles - These are flags that enable / disable features on a
    per customer or per environment basis and are not considered "temporary".
 
 ## Guide
 
-- For release toggles, prefer database backed feature flags
+- For release toggles, prefer database backed feature flags created an managed in Adminstrate -> Feature Flags
```

Comment:
Probably adding a comment saying we could use data migrations to automate the creation of the ffs 
https://github.com/BuoySoftware/guides/pull/97

Thank you!